src/Hadolint/Rule/DL3047.hs

```diff
@@ -0,0 +1,35 @@
+{-# LANGUAGE OverloadedStrings #-}
```

Comment:
You may remove all the language pragmas from `.hs` files as they are now globally enabled in `package.yaml`

Reply:
This one is totally just a convenience thing for me, as my editor currently does not recognize global pragmas, but I think this may be a configuration issue on my end.

Reply:
curious, what editor are you using?

Reply:
Vim with just a little bit of customization: I run a few auto commands, that remove trailing whitespaces from python sources and the likes, I also have `Ale` for running various linters and code checkers, and customized colors.
This particular problem was solvable by disabling everything in `Ale` that is not aware of either `package.yaml` or `*.cabal` files.

Reply:
Have you tried the haskell language server? I'm using it with (neo)vim and I'm extremely happy with it.